aws/sdk/examples/ebs/src/bin/create-snapshot.rs

```diff
@@ -52,27 +102,15 @@ async fn main() -> Result<(), aws_sdk_ebs::Error> {
             .send()
             .await?;
     }
-    let rsp = client
+    client
         .complete_snapshot()
         .changed_blocks_count(2)
         .snapshot_id(&snapshot_id)
         .send()
         .await?;
-    println!("snapshot complete: {:#?}", rsp);
-
-    // NOTE: you need to wait for `status != pending`
-    let ec2_client = aws_sdk_ec2::Client::from_env();
```

Comment:
can you add a reference to your `get-snapshot-state` example?

Reply:
Will do.